proxies/live/apiproxy/resources/jsc/ConnectionMetadata.SetRequestHeaders.js

```diff
@@ -31,7 +31,10 @@
   }
 
   var nrlAppID = context.getVariable("app.nrl-app-id");
```

Comment:
I'm guessing this doesn't fail if there is no variable and it only retrurns null or empty?